app/package.json

```diff
@@ -9,6 +9,7 @@
   },
   "repository": "zeit/hyper",
   "description": "HTML/JS/CSS Terminal",
+  "xo": false,
```

Comment:
This makes `xo` look upwards in the directory hierarchy for the next `package.json`, see https://github.com/sindresorhus/xo/pull/151.